cli/update.go

```diff
@@ -44,11 +44,11 @@ func (uc *UpdateCommand) addFlags() {
 	flagSet.Uint16Var(&uc.blkioWeight, "blkio-weight", 0, "Block IO (relative weight), between 10 and 1000, or 0 to disable")
 	flagSet.Int64Var(&uc.cpuperiod, "cpu-period", 0, "Limit CPU CFS (Completely Fair Scheduler) period, range is in [1000(1ms),1000000(1s)]")
 	flagSet.Int64Var(&uc.cpushare, "cpu-share", 0, "CPU shares (relative weight)")
+	flagSet.Int64Var(&uc.cpuquota, "cpu-quota", 0, "Limit CPU CFS (Completely Fair Scheduler) quota")
 	flagSet.StringVar(&uc.cpusetcpus, "cpuset-cpus", "", "CPUs in cpuset")
 	flagSet.StringVar(&uc.cpusetmems, "cpuset-mems", "", "MEMs in cpuset")
 	flagSet.StringVarP(&uc.memory, "memory", "m", "", "Container memory limit")
 	flagSet.StringVar(&uc.memorySwap, "memory-swap", "", "Container swap limit")
-	flagSet.Int64Var(&uc.memorySwappiness, "memory-swappiness", -1, "Container memory swappiness [0, 100]")
```

Comment:
I think memory-swappiness is not supported in update ops, but we ca still support it in create container. WDYT, @HusterWan

Reply:
Oh, sorry, I take it for create config, you are right.

Reply:
of course, we support set memory-swappiness when create container  !